app/pdfs/ticket_pdf.rb

```diff
@@ -49,18 +49,22 @@ def draw_second_square
     if @conference.picture?
       conference_image = case @conference.picture.ticket.url[0, 4]
                          when 'http', 'ftp:' # CDNs
-                           open(@conference.picture.ticket.url)
+                           begin
+                             open(@conference.picture.ticket.url)
+                           rescue OpenURI::HTTPError
+                             nil
+                           end
```

Comment:
Why do you explicitly need to return `nil`?

Reply:
Well, the `nil` return isn't explicitly necessary, but the rescue block has to have _some content_, and doing an explicit return is more elegant than, say `# awkward blank rescue block`( https://stackoverflow.com/questions/3672445/having-a-ruby-block-command-silently-fail-without-a-blank-rescue-block )